### ISSUE TITLE
Make the export and submit page more understandable

### ISSUE THREAD

**Issue by beparticular** [CMSgov/eAPD#2167]
_Reference enabler story #2194_ 

-The export and submit page seems to command many instructions, and isn't fully understandable or accurate
-The export/submit main page says "On the next page, you will be able to review and download a copy of your APD as the first step in submitting" but then the submit info is on that "first step" page, which is not a first step.
-The only way to submit is to go into that review page. 
-This page seems to encompass, Review, Download, and Submit
-Why do we use download and export interchangeably? 

### This task is done when...
 - [x] Create designs with consistent terms that help the user understand how and why we want them to review, download, and submit their APD
 - [x] Consider whether submit should be called out on the nav
 - [x] Consider whether submit belongs on the current "export and submit" page or if it's ok on the "review" 
 - [x] Change the title on the nav to match whatever decisions are made 
 - [x] We've produced mockups and/or prototype if needed of the design changes
 - [x] Product has approved designs

**Comment by nicholeweems** [comment]
Will be discussed as part of the update experience discussion. Will revisit user story post discussion.

**Comment by jjames521** [comment]
@beparticular 

Just as a quick initial assumptions/basis mockup, before we start thinking about it further in depth.

Some of the things I considered,

Review as a simple nav header to replace submit and export acting as a lead to push the user to consider looking over the summary more in depth.  The submit and export are predominantly presented first anyway as the user clicks the button to review. 

I thought, maybe, we could get rid of the page all together and consolidate but leading the user through the current journey allows for the page to be full scale and doesn't distract the user from reviewing.

Changed the wording from _"Continue to Review"_, to _"Proceed to Review"_ for clarity.


![ReviewExportSubmitVariant.png](https://images.zenhubusercontent.com/5e7151677c78c820a2a11970/ba7d3a2b-76bc-42eb-ac0f-a46769ef7d17)

**Comment by CSwartzHMA** [comment]
I also feel like its a bit clunky to have this page that is really just a pass through to the next page, but it is consistent with the design and I don't think we have had any user feedback to indicate they are confused by this page. it does just seem like extra clicks.  I am wondering if there is way to rework the export and submit page so that we could eliminate the jump page.

**Comment by beparticular** [comment]
@CSwartzHMA I'd like us to explore that at some point, but think it's out of scope for this particular issue. 
Jesse and I spent some time on this one last week and have some directions to work in. We can talk about it more in design sync tomorrow.

**Comment by jjames521** [comment]
@beparticular 
@CSwartzHMA 

Change log:
Moved "Blue box" contents out of review page onto Export and Submit area.
Changed to Review and Export in nav and appropriate titles.
Added in divider lines to help delineate between sections
Removed "first" from descriptions
Removed "download" and changed to export for consistency.

![ReviewExportSubmitVariant1.png](https://images.zenhubusercontent.com/5e7151677c78c820a2a11970/bb6ca631-5968-4927-8c95-d7b22f486d5d)

**Comment by jjames521** [comment]
**Change log:**
Moved "Blue box" contents out of review page onto Export and Submit area.
Changed to Review and Export in nav and appropriate titles.
Added in divider lines to help delineate between sections.
Removed "first" from descriptions.
Removed "download" and changed to export for consistency.
Added "Back to APD" link at end of export view.
Changed sequence in mockup to review, export, and submit.

![ReviewExportSubmitVariant1.png](https://images.zenhubusercontent.com/5e7151677c78c820a2a11970/df91d038-d2c7-4d7a-a078-3113f61aa699)


![ReviewWithoutExportSubmit.png](https://images.zenhubusercontent.com/5e7151677c78c820a2a11970/eab54b7a-7006-4cb8-95b7-985774bbcd5f)

![ReviewExportBacktoAPDLink.png](https://images.zenhubusercontent.com/5e7151677c78c820a2a11970/bbfefa83-54be-4395-9453-297cde5a5ef5)

**Comment by beparticular** [comment]
When we do get to implementing this - take into consideration the design changes made to headers and lines on the top of pages in the app. #2370

**Comment by thetif** [comment]
OBE
